pkg/ignition.go

```diff
@@ -69,6 +82,33 @@ func IgnitionFromOperatingSystemConfig(ctx context.Context, c client.Client, con
 		cfg.Storage.Files = append(cfg.Storage.Files, ignitionFile)
 	}
 
+	if config.Spec.CRIConfig != nil {
+		cri := config.Spec.CRIConfig
+		if cri.Name == extensionsv1alpha1.CRINameContainerD {
+
+			containerdSystemdService := types.SystemdUnit{
+				Name: "containerd.service",
+				Dropins: []types.SystemdUnitDropIn{
+					{
+						Name:     "11-exec_config.conf",
+						Contents: containerdSystemdDropin,
+					},
+				},
+			}
+			cfg.Systemd.Units = append(cfg.Systemd.Units, containerdSystemdService)
+
+			containerdConfigFile := types.File{
```

Comment:
This would create the file `/etc/containerd/config.toml` ?.
Unfortunately the Gardener script only checks for file existence , and if there is no such file, generates the default config.

In this case, the file would exist, Gardener does not create the default config, leaving containerd with an incorrect config file.

Reply:
But you create a default config in your script, this has the same effect as an empty config. This what i did, why do you expect it is a wrong config ? What specific config options do you require ?

Reply:
I think you are right, the empty config will have the same effect. 

The only thing is that if Gardener in the future allows the configuration of containerd, the code should be adjusted. Is there a reason that you do not want Gardener to create the config.toml?